packages/perseus-editor/src/widgets/__tests__/interactive-graph-editor-locked-figures.test.tsx

```diff
@@ -303,6 +305,7 @@ describe("InteractiveGraphEditor locked figures", () => {
         test("Calls onChange when a locked line is removed", async () => {
             // Arrange
             const onChangeMock = jest.fn();
+            window.confirm = jest.fn(() => true); // always click yes
```

Comment:
Is it possible to use `const confirmSpy = jest.spyOn(window, "confirm")` instead?   That way we don't have to worry about resetting the `window.confirm` function manually after tests run (Jest does that for us)